Stack/Opc.Ua.Core/Security/Certificates/CertificateFactory.cs

```diff
@@ -226,7 +226,8 @@ public static X509Certificate2 Load(X509Certificate2 certificate, bool ensurePri
         ushort lifetimeInMonths,
         ushort hashSizeInBits,
         bool isCA,
-        X509Certificate2 issuerCAKeyCert)
+        X509Certificate2 issuerCAKeyCert,
+        byte[] publicKey = null)
     {
         if (issuerCAKeyCert != null)
         {
```

Comment:
Nope, you can also sign a cert with an existing public key not meant as a CA cert.